modules/insights-operator-configuring.adoc

```diff
@@ -19,28 +19,111 @@ The `ConfigMap` object does not exist by default, so an {product-title} cluster
 
 image::insights-operator-configmap-example.png[Example of Insights Operator ConfigMap object]
 
 .Configurable attributes and default values
-The table below describes the available configuration attributes:
+The following table describes the available configuration attributes:
 
 [NOTE]
 ====
 The *insights-config* `ConfigMap` object follows standard YAML formatting, wherein child values are below the parent attribute and indented two spaces. For the *Obfuscation* attribute, enter values as bulleted children of the parent attribute.
 ====
 
 .Insights Operator configurable attributes
-[options="header"]
+[cols=".^2l,.^3a,.^1a,.^1a",options="header"]
 |====
 |Attribute name|Description|Value type|Default value
-|`Obfuscation: - networking`|Enables the global obfuscation of IP addresses and the cluster domain name.|Boolean|`false`
-|`Obfuscation: - workload_names`|Obfuscate data coming from the Deployment Validation Operator if it is installed.|Boolean|`false`
-|`sca: interval`|Specifies the frequency of the simple content access entitlements download.|Time interval|`8h`
-|`sca: disabled`|Disables the simple content access entitlements download.|Boolean|`false`
-|`alerting: disabled`|Disables Insights Operator alerts to the cluster Prometheus instance.|Boolean|`false`
-|`httpProxy`, `httpsProxy`, `noProxy`|Set custom proxy for Insights Operator|URL|No default
+
+|alerting: 
+    disabled: false
+|Disables Insights Operator alerts to the cluster Prometheus instance.
+|Boolean
+|`false`
+
+|clusterTransfer: 
+    endpoint: <url>
+|The endpoint for checking and downloading cluster transfer data.
+|URL
+|https://api.openshift.com/api/accounts_mgmt/v1/cluster_transfers/
+
+|clusterTransfer: 
+    interval: 1h0m0s
+|Sets the frequency for checking available cluster transfers.
+|Time interval
+|24h
+
+|dataReporting:
+    interval: 30m0s
+|Sets the data gathering and upload frequency.
+|Time interval
+|2h
+
+|dataReporting:
+    uploadEndpoint: <url>
+|Sets the upload endpoint.
+|URL
+|https://console.redhat.com/api/ingress/v1/upload
+
+|dataReporting:
+    storagePath: <path>
+|Configures the path where archived data gets stored.
+|File path
+|/var/lib/insights-operator
+
+|dataReporting:
+    downloadEndpoint: <url>
+|Specifies the endpoint for downloading the latest Insights analysis.
+|URL
+|https://console.redhat.com/api/ingress/v1/download
+
+|dataReporting:
+    conditionalGathererEndpoint: <url>
+|Sets the endpoint for providing conditional gathering rule definitions.
+|URL
+|https://console.redhat.com/api/gathering/gathering_rules
+
+
+|dataReporting:
+    obfuscation:
+    - networking
+|Enables the global obfuscation of IP addresses and the cluster domain name.
+|Boolean
+|`false`
+
+|dataReporting:
+    obfuscation:
+    - workload_names
+|Obfuscate workload names from resource user IDs (UIDs).
```

Comment:
I would probably mention that it obfuscates only the data produced by Data Validation Operator. By default for the cluster resources there is a `name` + `id` included, when this is configured then the name is removed.